app/controllers/admin/client_users_controller.rb

```diff
@@ -6,7 +6,7 @@ class ClientUsersController < Admin::BaseController
     def index
 
       @page_title = "Client Users | Admin"
-
+      @client_users = ClientUser.all
```

Comment:
Please implement pagination
you can use pagination helper methods I have added to helpers/
just try ClientUser.per(@current_page).page(@per_page)
current_page and per_page will be autopopulated as @ variables if those methods are called in before action

Reply:
implemented.